Dockerfile

```diff
@@ -2,6 +2,13 @@ FROM python:3-alpine
 
 LABEL maintainer="CloudOps <cloudops@clark.de>"
 
+# Install kubectl
+RUN apk add --no-cache openssl curl tar gzip bash ca-certificates
```

Comment:
we could add these to the other RUN to avoid extra layers

Reply:
```
RUN \
  apk add --no-cache --upgrade git openssl curl tar gzip bash ca-certificates && \
  curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && \
  chmod +x /usr/bin/kubectl && \
  kubectl version --client
  python3 /app/setup.py install && \
  apk del git && \
  rm -rf /app /var/cache/apk
```

Reply:
done